transform/util.go

```diff
@@ -133,6 +129,82 @@ func getJSONRaw(data []byte, path string, pathRequired bool) ([]byte, error) {
 	return result, nil
 }
 
+// setPath updates the value with properly formatted timestamp(s) and properly
```

Comment:
Update comment to proper function name and maybe don't reference the timestamp explicitly?